.github/workflows/main.yml

```diff
@@ -31,7 +31,7 @@ env:
   # /dev.Dockerfile
   # /make/builder.Dockerfile
   # /.github/workflows/release.yml
-  GO_VERSION: 1.22.3
+  GO_VERSION: 1.22.5
```

Comment:
Should we remove `/.travis.yml` from the list in all these files, since we don't use it anymore?

Reply:
Yeh SGTM, tried a few diff grep commands until I convinced myself that it actually wasn't there lol